content/applications/websites/website/optimize/plausible.rst

```diff
@@ -1,42 +1,44 @@
+===================================
+Website analytics with Plausible.io
+===================================
 
 To follow your website's traffic with Plausible.io:
 
-- `Create a Plausible.io account <https://plausible.io/register/>`__ if you don't have any.
+- `Create a Plausible.io account <https://plausible.io/register>`_ if you don't have any.
+- `Create your website <https://plausible.io/sites/new>`_ and `Start collecting data` if it is not
+  already done.
+- Go through the `list of your websites <https://plausible.io/sites>`_ and click on the gear icon.
 
-- `Create your website <https://plausible.io/sites/new/>`__ and `Start collecting data` if it is not already done.
+  .. image:: plausible/cog.png
+     :align: center
+     :alt: Click on cog icon in the list of websites
 
-- Go through the `list of your websites <https://plausible.io/sites/>`__ and click on the gear icon.
+- Click on the button :guilabel:`+ New Link` in the tab Visibility to create your Shared links.
 
-    .. image:: plausible/plausible_cog.png
-       :align: center
+  .. image:: plausible/new-sharedlink.png
+     :align: center
+     :alt: Adding a shared link in the Visibility tab
 
-- Click on the button `+ New Link` in the tab Visibility to create your Shared links.
+- Choose a name (and NO password - it is not supported to embed it in Odoo) and click on the button
+  `Create Shared Link`.
 
-    .. image:: plausible/plausible_new_sharedlink.png
-       :align: center
+  .. image:: plausible/create-sharedlink.png
+     :align: center
+     :alt: Credentials creation for the new shared link
 
-- Choose a name (and NO password - it is not supported to embed it in Odoo) and click on the button `Create Shared Link`
+- Copy the new Shared Link URL from Plausible.io and paste it into the Settings of Odoo.
 
-    .. image:: plausible/plausible_create_sharedlink.png
-       :align: center
+  .. image:: plausible/copy-sharedlink.png
+     :align: center
+     :alt: Copy the shared link URL from Plausible.io
 
-- Copy the new Shared Link url from Plausible.io and paste it in the Settings of Odoo
+  .. image:: plausible/paste-sharedlink.png
+     :align: center
+     :alt: Paste the shared link URL to Odoo settings
 
-    .. image:: plausible/plausible_copy_sharedlink.png
-       :align: center
-    .. image:: plausible/plausible_paste_sharedlink.png
-       :align: center
-
-
-.. note:: You are on the Odoo Saas Platform?
-
-    When you `create your database on www.odoo.com/trial <https://www.odoo.com/trial>`__, we create it for you on our own server.
-    You don't have anything to do.
-
-
-.. note:: What about the Goal?
-
-    Odoo push out-of-the-box some custom event: `Lead Generation` and `Shop`.
+.. note::
+   - | **You are on the Odoo Online Platform**
+     | When you `create your Odoo Online database <https://www.odoo.com/trial>`_, we create it for
+       you on our own server. You don't have anything to do.
```

Comment:
If i understand correctly, plausible is automatically setup on odoo online databases ?  Shouldn't it be said in the beginning of the page then ?

Reply:
@Feyensv 
I mostly fixed the technical aspect of the page ("fix" commit), not the content. Otherwise, I'd rewrite it ^^